CHANGES.rst

```diff
@@ -1,7 +1,8 @@
 0.4.1 (unreleased)
 ==================
 
-- No changes yet.
+- Reverting the short option of ``-R`` due to a clash with ``pytest-leaks``.
+  The short option is added to ``pytest-astropy`` instead. [#70]
```

Comment:
But won't that make any package that install `pytest-astropy` to be incompatible with `pytest-leaks`?

Reply:
Yes, it would. But -R was around in the astropy ecosystem forever, so I don't expect much clash in real life scenarios.

Reply:
I guess it is a nice compromise. Packages don't really need the meta package to function if there is indeed a clash down the road.

Reply:
What is the order of operation? Does https://github.com/astropy/pytest-astropy/pull/55 go in first or this one?